demos/CompositeUI-MVC/Divergent.ITOps.ViewModelComposition/DynamicViewModel.cs

```diff
@@ -77,9 +84,28 @@ public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, o
             return false;
         }
 
+        public override IEnumerable<string> GetDynamicMemberNames()
+        {
+            return base.GetDynamicMemberNames()
+                .Union(properties.Keys)
+                .Union(new[] { "RaiseEventAsync", "RaiseEvent" });
```

Comment:
You forgot `yield return "RaiseEvent";` but yes. Unless it would be normal for there to be overlap that would cause an issue? Then you could probably maintain your own HashSet cheaper than calling Union twice. (Although that's a guess.)